content/applications/finance/accounting/customer_invoices.rst

```diff
@@ -4,225 +4,221 @@
+and expenses.
 
-.. image:: customer_invoices/invoice12.png
+Balance sheet
+-------------
 
-.. _customer-invoices/aging-report:
+The :ref:`Balance Sheet <accounting/reporting/balance-sheet>` summarizes the company's liabilities,
+assets, and equity at a specific time.
 
-Customer aging report:
-~~~~~~~~~~~~~~~~~~~~~~
+.. _accounting/invoice/resequencing:
 
-The customer aging report will be an additional key tool for the
-collector to understand the customer credit issues, and to prioritize
-their work.
+Resequencing
+============
 
-Use the aging report to determine which customers are overdue and begin
-your collection efforts.
+:guilabel:`Resequence` is a technical feature used to change the sequence of documents. It can be
+helpful to resequence invoice numbers when importing invoices from another invoicing or accounting
+system, mainly if the reference originates from the previous software and continuity for the current
+year must be maintained without restarting from the beginning.
 
-.. image:: customer_invoices/invoice13.png
+The sequence can be modified once an invoice is in the system.
 
-Profit and loss
----------------
+.. note::
 
-The Profit and Loss statement displays your revenue and expense details.
-Ultimately, this gives you a clear image of your Net Profit and Loss. It
-is sometimes referred to as the "Income Statement" or "Statement of
-Revenues and Expenses."
+   - This feature is only available to users with administrator or advisor access.
+   - All sequence changes are logged in the chatter to keep the information.
 
-.. image:: customer_invoices/invoice14.png
+Here are the different steps to resequence invoice numbers:
 
-Balance sheet
--------------
+#. Activate the :ref:`developer mode <developer-mode>`.
+#. From the :guilabel:`Accounting Dashboard`, open the :guilabel:`Customer Invoices` journal.
+#. Select the documents that need a new sequence.
+#. In the :icon:`fa-cog` :guilabel:`Actions` menu, click :guilabel:`Resequence`.
+#. In the :guilabel:`Ordering` field, choose to
 
-The balance sheet summarizes your company's liabilities,
-assets and equity at a specific moment in time.
+   - :guilabel:`Keep current order`: the order of the numbers remains the same
+   - :guilabel:`Reorder by accounting date`: the number is reordered by accounting date.
+#. Set the :guilabel:`First New Sequence`.
+#. :guilabel:`Preview Modifications` and click :guilabel:`Confirm`.
 
-.. image:: customer_invoices/invoice15.png
+.. image:: customer_invoices/invoice-sequencing.png
+   :alt: Resequence options window
 
-.. image:: customer_invoices/invoice16.png
+In some cases, resequencing is not possible:
 
-For example, if you manage your inventory using the perpetual accounting
-method, you should expect a decrease in account "Current Assets" once
-the material has been shipped to the customer.
+- When entries are previous to a lock date.
+- When changing the date of a previously posted invoice from another period, the sequence is not
```

Comment:
Can we get an example for this one so it's more clear?

Reply:
I also changed the text to make it more clear.